src/Fuel-Tools-Pharo/FuelOutStackDebugAction.class.st

```diff
@@ -95,7 +95,7 @@ FuelOutStackDebugAction >> serializeStack [
 	fileName := String streamContents: [ :s|
 		s 
 			<< 'Debugger-Stack-';
-			<< self interruptedContext receiver class instanceSide name;
+			<< self interruptedContext receiver class theNonMetaClass name;
```

Comment:
Oh, I missed this one. It should be #instanceSide too I think. Probably a change not ported to Fuel repository.

Reply:
Thanks. I'll fix that too.